packages/actions/test/index.test.ts

```diff
@@ -0,0 +1,67 @@
+import { httpsCallable } from "firebase/functions"
+import chai, { assert } from "chai"
+import chaiAsPromised from "chai-as-promised"
+import { FirebaseDocumentInfo } from "types"
+import { getOpenedCeremonies } from "../src/index"
+import { initializeAdminServices, initializeUserServices, signInAnonymouslyWithUser, deleteAdminApp } from "./utils"
+import { fakeUsersData, fakeCeremoniesData, fakeCircuitsData } from "./data/samples"
+
+// Config chai.
+chai.use(chaiAsPromised)
+
+describe("Sample e2e", () => {
+    // Sample data for running the test.
+    let openedCeremonies: Array<FirebaseDocumentInfo> = []
+    let selectedCeremony: FirebaseDocumentInfo
+
+    // Initialize admin and user services.
+    const { adminFirestore } = initializeAdminServices()
+    const { userApp, userFirestore, userFunctions } = initializeUserServices()
+
+    beforeEach(async () => {
+        // Sign-in anonymously with the user.
+        await signInAnonymouslyWithUser(userApp)
```

Comment:
It stores testing user data into remote datastore but didn't removed after test has ended. Here is an example remaining user data:
<img width="581" alt="Screenshot 2022-12-04 at 11 18 35" src="https://user-images.githubusercontent.com/1610146/205485355-92ff7173-f56a-425a-aa3f-59254f85c7eb.png">

Reply:
Fixed here #225